### ISSUE TITLE
Jetpack Stats show percentages in the Emails section

### ISSUE THREAD

**Issue by ariel-maidana** [Automattic/wp-calypso#97812]
### Quick summary

In the Email section of the Subscribers tab of Jetpack Stats (https://wordpress.com/stats/subscribers/) opens and clicks are shown as percentages instead of absolute numbers. The example in the [support page](https://wordpress.com/support/stats/understand-your-sites-traffic/#emails) shows absolute numbers.

![Image](https://github.com/user-attachments/assets/dc8c485e-3441-429a-9dc5-e4459b859643)

### Steps to reproduce

1. On any site with a newsletter and active subscribers, go to https://wordpress.com/stats/subscribers/site_url
2. Scroll down to the Emails section. 
3. See issue.

### What you expected to happen

Clicks and opens should be shown in absolute numbers, as in the documentation (https://wordpress.com/support/stats/understand-your-sites-traffic/#emails).

![Image](https://github.com/user-attachments/assets/edb3f27d-3445-41db-b97d-7584aaf2b994)

### What actually happened

Clicks and opens are shown in percentages.

### Impact

All

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Simple, Atomic

### Logs or notes

Context: 9179725-zd-a8c
Related ticket: https://github.com/Automattic/wp-calypso/issues/97601

**Comment by valterlorran** [comment]
I redirected this to @Automattic/zap as they changed the behavior. I'm unsure of the exact motivation, but I believe they can enlighten us.

https://github.com/Automattic/wp-calypso/pull/97581
https://github.com/Automattic/jetpack/issues/40538

If this is the new behavior to be, we should update the docs as listed by @ariel-maidana.

**Comment by lezama** [comment]
Good catch! Yes, please! Could you update the docs? We will be showing rates instead of clicks going forward. You can view the details in the tooltip or on the individual email page. We made this change based on feedback from newsletter owners, as they find rates more useful than clicks.

<img width="273" alt="Image" src="https://github.com/user-attachments/assets/75e6db8b-48df-4289-a215-5ee0ed92463e" />

**Comment by lezama** [comment]
Tooltip will show up after we merge https://github.com/Automattic/wp-calypso/pull/97729

**Comment by valterlorran** [comment]
Hey, @lezama, thank you very much for the context. Can you please provide me with a good screenshot so I can update the docs? Feel free to DM me on Slack to avoid losing quality.

**Comment by valterlorran** [comment]
Closing this as we updated the docs to reflect the changes.

More context: p1735844843288229-slack-CFS7FQ9FD
